app/MMB/src/scenes/tripServices/insurance/InsuranceTermsPdfScene.js

```diff
@@ -0,0 +1,23 @@
+// @flow strict
+
+import * as React from 'react';
+import { DeviceInfo } from '@kiwicom/mobile-localization';
+import { PdfViewAndStore } from '@kiwicom/mobile-pdf';
+
+const InsuranceTermsPdfScene = () => {
+  const languageCode = DeviceInfo.getLanguage();
+  let url =
+    'https://s3-eu-west-1.amazonaws.com/skypicker-pdf/insurance/VPPCP-Kiwi.com_ENG-CZE.pdf';
+  if (languageCode === 'cs') {
+    url =
+      'https://s3-eu-west-1.amazonaws.com/skypicker-pdf/insurance/VPPCP-Kiwi.com_CZE-ENG.pdf';
+  }
+  return (
+    <PdfViewAndStore
+      fileName={`/terms-and-conditions/insurance-${languageCode}.pdf`}
+      url={url}
+    />
```

Comment:
Not sure we want to store this pdf to the users device. We have a component also for viewing, where you can cache it.

Reply:
Ok, I will replace it.